### ISSUE TITLE
cockpit view problem

### ISSUE THREAD

**Issue by tommaso21** [heclak/community-a4e-c#598]
I installed the A-4E mod last revision and is working fine. Unfortunatly when I click F1 the screen becomes black .. what could be the reason ? Thanks

**Comment by callmepartario** [comment]
there are apparently a very small percentage of users who have some sort of video driver issue in the cockpit that causes this, but we've never been able to correctly identify the issue.

**Comment by kariella** [comment]
I just downloaded and installed it and am having this issue... I tried to comment out the radar warning line but that didnt help

**Comment by kariella** [comment]
yes Dell Optiplex 3050 with a integrated Intel HD Graphics 630 - (I am looking for a vid card for it)... its just a weird problem since it only involves one plane and one specific spot in the cockpit... I believe its where the radar is... rest of the cockpit is fine...
well if its something you are looking to troubleshoot and there is any data I can send you plmk...

**Comment by callmepartario** [comment]
that's good confirmation for this issue. have you tried going into the A-4E-C special menu and checking the box to use the old radar code instead?

**Comment by 3DaniToJa** [comment]
Seems that turning on the legacy radar didnt help. I'll see if rebooting DCS will help. I've seen a hot fix earlier that helped someone before. It worked for him but not for me. The hot fix was to add "--" before RADARWARN system in the cockpits LUA script.

**Comment by 3DaniToJa** [comment]
Turns out that with the legacy screen I can see more of the cockpit! However the issue still remains. I can see more of the screen but it seems that theres one model in the middle of it that makes the screen go null. I can see most of the pannel but it's monstly from the right side, whenever I tried seeing it on the left how far I can go it went black nearly instantly. I suspect some object on the left side of the pannel or the white knob/part of the screen frame in the middle.
![20241226175239_1](https://github.com/user-attachments/assets/8feff82e-b1c8-4d2f-84a1-9f3f05a2265e)
![20241226175222_1](https://github.com/user-attachments/assets/bed776fc-0b08-4d1b-97dc-db398cf48dda)
![20241226175157_1](https://github.com/user-attachments/assets/f0bbb9e3-7627-4647-8c68-11eabe639918)
![20241226175118_1](https://github.com/user-attachments/assets/b371ea72-b653-44d0-b526-ef7814463259)
here's pictures of how far I could go from left right top and bottom

**Comment by v0lkss** [comment]
So this problem has no solution?

**Comment by 08jne01** [comment]
> So this problem has no solution?

The workaround could be to disable the indicators for the radar completely. You could do this by changing
`indicators[#indicators + 1] = {"ccIndicator" ,            LockOn_Options.script_path.."RADAR/Indicator/init.lua"    ,                 nil, {{"PNT_RADAR_CENTER","PNT_RADAR_DOWN","PNT_RADAR_RIGHT"}}}`

to

`--indicators[#indicators + 1] = {"ccIndicator" ,            LockOn_Options.script_path.."RADAR/Indicator/init.lua"    ,                 nil, {{"PNT_RADAR_CENTER","PNT_RADAR_DOWN","PNT_RADAR_RIGHT"}}}`

here: https://github.com/Community-A-4E/community-a4e-c/blob/master/A-4E-C/Cockpit/Scripts/device_init.lua#L118

**Comment by callmepartario** [comment]
if this works, i could work on adding special menu option checkbox to do this. let us know.
